docs-website/generateDocsDir.ts

```diff
@@ -395,7 +395,7 @@ function markdown_sanitize_and_linkify(content: string): string {
 
 function pretty_format_date(datetime: string): string {
   const d = new Date(Date.parse(datetime));
-  return d.toDateString();
+  return d.toISOString().split("T")[0];
```

Comment:
WHy this one?

Reply:
It produces the format `2022-08-11`, which I think looks more readable than what we have right now `Tue Aug 09 2022`